_DeploymentAndDistroPackaging/ansible/cleanup/ciao.yml

```diff
@@ -17,16 +17,11 @@
   connection: local
   tasks:
     - name: Remove local files
-      local_action: file path={{ item }} state=absent
-      with_items:
-        - ./openrc
-        - ./admin-ciaorc
-        - ./demo-ciaorc
-        - ./clouds.yaml
-        - ./certificates
-        - ./images/.cnci_ready
-        - ./fetch
-        - /tmp/go
+      file: path={{ item }} state=absent
+      with_fileglob:
+        - /tmp/go/*
```

Comment:
ansible does not create a directory if the directory already exists. thats how the [file](http://docs.ansible.com/ansible/file_module.html) module works